frappe/core/doctype/access_log/access_log.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2019, Frappe Technologies and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('Access Log', {
+	show_document: function (frm) {
+		frappe.set_route('Form', frm.doc.export_from, frm.doc.reference_document);
+	},
+
+	show_report: function (frm) {
+		if (frm.doc.report_name.includes('/')) {
+			frappe.set_route(frm.doc.report_name);
+		} else {
+			let filters = frm.doc.filters ? frm.doc.filters : {};
```

Comment:
Are you sure this works? I think you need to do `JSON.parse` because `frm.doc.filters` is saved as a JSON string.

Reply:
writing tests makes a man do crazy things sometimes (like becoming a caffeine addict and forgetting to hard reload). 

Thanks for catching that!